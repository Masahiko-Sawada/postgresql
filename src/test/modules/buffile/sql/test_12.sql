SELECT buffile_test_shared();
SELECT buffile_test_shared_append();
