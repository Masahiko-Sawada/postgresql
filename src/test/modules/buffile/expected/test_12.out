SELECT buffile_test_shared();
 buffile_test_shared 
---------------------
 
(1 row)

SELECT buffile_test_shared_append();
 buffile_test_shared_append 
----------------------------
 
(1 row)

